src/aics-image-viewer/components/TfEditor/index.tsx

```diff
@@ -356,6 +360,9 @@ const TfEditor: React.FC<TfEditorProps> = (props) => {
       if (el === null) {
         return;
       }
+      if (props.channelData.histogram.getNumBins() < 1) {
+        return;
+      }
```

Comment:
the only change in this file is this one which is basically a "null check".. at some point during debugging this, there was a condition where empty channelData was coming in here.